corehq/db/config.yml

```diff
@@ -0,0 +1,20 @@
+# HQ database configuration for multiple database. This gets mapped to Django's DATABASES dict
```

Comment:
> - `settings.py` is 100% static/shared

By this, do you mean there will be no more `import localsettings` in `settings.py`? And no more if/then, for loops, etc. Just a simple python file in which the every line has the form `NAME = <expression>`.

> - all custom variables are accessed through `settings_manager`

And by this you mean `settings.py` looks like
```py
# simple: get settings manager value
DEBUG = settings_manager.get_variable("debug")

# dynamic: get settings manager calculated value
DATABASES = settings_manager.get_databases()

# settings that never change (not configurable)
TIME_ZONE = "UTC"

...
```
right?

> - all `settings_manager` custom variables get set through one or more static config files

So like we add a `/conf` entry to `.gitignore`, and all local settings get set in files like `conf/databases.yaml`. This `conf` dir is where `SettingsManger` looks for local configuration settings. Config files in `conf` are 100% optional. HQ should load sensible defaults for a newly hired developer if there are no files in `conf`.

If those clarifications sound right, then I think this seems like a great idea. Can I work on this?